src/applications/lgy/coe/containers/introductionContent/notLoggedInContent.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import SaveInProgressIntro from 'platform/forms/save-in-progress/SaveInProgressIntro';
+
+export const notLoggedInContent = props => (
+  <div>
+    <va-alert status="info">
+      <h3 slot="headline">You may be able to get an automatic COR online</h3>
```

Comment:
COR & COE defined somewhere on this page?

Reply:
Damn autocorrect. Fixed.